locales/sv.lua

```diff
@@ -35,7 +35,10 @@ local Translations = {
     }
 }
 
-Lang = Locale:new({
-    phrases = Translations,
-    warnOnMissing = true
-})
+if GetConvar('qb_locale', 'en') == 'sv then
```

Comment:
```suggestion
if GetConvar('qb_locale', 'en') == 'sv' then
```
Finish the string here